### PR TITLE
Makes nukeop start with syndie gasmask in internal box

### PR DESCRIPTION
**Pull request by Bawhoppen**
why noone did this sooner I do not know

### CODE REVIEW

**Comment by PKPenguin321** [comment]
:+1:

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -77,6 +77,15 @@
 	new /obj/item/weapon/reagent_containers/hypospray/medipen(src)
 	return
 
+/obj/item/weapon/storage/box/engineer/syndie
+
+/obj/item/weapon/storage/box/engineer/syndie/New()
+	..()
+	contents = list()
```
Comment (phil235) [inline code/game/objects/items/weapons/storage/boxes.dm]: why not /obj/item/weapon/storage/box/syndie? so you don't have to do that hacky stuff.

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 2]
```diff
@@ -77,6 +77,15 @@
...
 	return
 
+/obj/item/weapon/storage/box/engineer/syndie
+
+/obj/item/weapon/storage/box/engineer/syndie/New()
+	..()
+	contents = list()
+	new /obj/item/clothing/mask/gas/syndicate(src)
+	new /obj/item/weapon/tank/internals/emergency_oxygen/engi(src)
+	return
```
Comment (RemieRichards) [inline code/game/objects/items/weapons/storage/boxes.dm]: trailing return, remove it please.

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 1 above)
Comment (Bawhoppen) [inline code/game/objects/items/weapons/storage/boxes.dm]: I guess it didn't seem that important oh well

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 2 above)
Comment (Bawhoppen) [inline code/game/objects/items/weapons/storage/boxes.dm]: ok

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 3]
```diff
@@ -75,7 +74,14 @@
 	new /obj/item/clothing/mask/breath(src)
 	new /obj/item/weapon/tank/internals/emergency_oxygen/engi(src)
 	new /obj/item/weapon/reagent_containers/hypospray/medipen(src)
-	return
+
+/obj/item/weapon/storage/box/syndie
+
+/obj/item/weapon/storage/box/syndie/New()
+	..()
+	contents = list()
```
Comment (KorPhaeron) [inline code/game/objects/items/weapons/storage/boxes.dm]: You do not need this line

Comment (coiax) [inline code/game/objects/items/weapons/storage/boxes.dm]: Oh, is that the proper way of emptying out an inherited container?

Comment (Bawhoppen) [inline code/game/objects/items/weapons/storage/boxes.dm]: I literally just followed the conventions of every other one

**Pull request merged**